data/json/monstergroups/zombies.json

```diff
@@ -91,6 +91,58 @@
       { "monster": "mon_zombie_scorched_brute", "weight": 5, "cost_multiplier": 5 }
     ]
   },
+  {
+    "type": "monstergroup",
+    "id": "GROUP_ZOMBIE_HORDE",
```

Comment:
Once we have different behaviors for different horde entities we can re-add panicked persons to these groups or go back to using the existing group, but that's too much scope creep for this PR.